test/relative-artifacts.test.ts

```diff
@@ -0,0 +1,24 @@
+import { expect } from "chai";
+import { starknet } from "hardhat";
+import { StarknetContract, StarknetContractFactory } from "hardhat/types/runtime";
+import { TIMEOUT } from "./constants";
+
+describe("Starknet", function () {
+    this.timeout(TIMEOUT);
+
+    it("quick test for relative artifacts", async function () {
+        console.log("Started deployment");
+        const contractFactory: StarknetContractFactory = await starknet.getContractFactory("../test/test-artifacts/contract");
+        const contract: StarknetContract = await contractFactory.deploy({ initial_balance: 0 });
+        console.log("Deployed at", contract.address);
+
+        const { res: balanceBefore } = await contract.call("get_balance");
+        expect(balanceBefore).to.deep.equal(0n);
+
+        await contract.invoke("increase_balance", { amount1: 10, amount2: 20 });
```

Comment:
What do we get from running the increase_balance and get_balance? The purpose of this test is to test if the contract factory can be loaded from a relative path, which is sufficiently tested with lines 10-16 (so we can keep the first get_balance, but remove the increase_balance and remove the second get_balance).

Reply:
Well, I wanted not only check load but also that it deploys and functions properly. But you're right lines 10-16 shall be enough. Removed other calls